### PR TITLE
Add new ED to website

### PR DESCRIPTION
**Pull request by david-christiansen**
This re-enables the "Executive team" section and adds me and my picture to it.

The existing pictures are not entirely consistent - some have lower resolution and a white circle, others are larger with a transparent border around the circle. I opted for the latter in the interests of flexibility, but in case I missed some commonality, I'm happy to modify it.

### CODE REVIEW

**Comment by david-christiansen** [comment]
I don't understand the CI failure here. It looks like it's attempting to deploy the site somewhere, but that it's not doing so. But there's no error message.

Any suggestions on where to start diagnosing the issue?

File: who-we-are/people/davidchristiansen.markdown [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: David Thrane Christiansen
+email: david@haskell.foundation
+title: Executive Director
+executiveTeam: "True"
+tenureStart: 2022-05-02
+image: /assets/images/exec-team/dtc.png
+---
```
Comment (goldfirere) [inline who-we-are/people/davidchristiansen.markdown:8]: You mentioned to me elsewhere that you might want a link to your website nearby. You can add one with a `website: https://blah/bah` metadata item here.

**Review by goldfirere — COMMENTED** [review]
One possible oversight; otherwise good. You're right that the images are non-uniform, something I discovered recently, too. I think this is historical accident. Your picture renders well on my screen.

**Comment by goldfirere** [comment]
The CI failure is expected. (I'm not sure how all the belts and gears work here.) To test, `stack exec -- site build` or `cabal run -- site build` locally, and then examine `_site/index.html`. When you merge, deployment is automatic and appears to work reliably.

For what it's worth, I tested this on my own machine, and all looks good.

Fixing the CI failure on branches appears to be the subject of #170, but I have not looked at this myself.

File: who-we-are/people/davidchristiansen.markdown (hunk 1 above)
Comment (david-christiansen) [reply who-we-are/people/davidchristiansen.markdown:9]: Ah, I figured it out - it needed a quick tweak of the template as well. Thanks!

**Comment by david-christiansen** [comment]
It looks good with a local test on my end too.

**Review by goldfirere — APPROVED** [review]

**Comment by goldfirere** [comment]
I took the liberty to merge. In general, I'm unaware of a specific protocol for who is supposed to merge for these PRs -- feel free to set down your own desired workflow here. :)

**Comment by david-christiansen** [comment]
I generally like workflows where people don't merge their own PRs, assuming there's sufficient collaborators to make that practicable. Thanks!

**Pull request merged**